### PR TITLE
OCP 4.4

### PR DESCRIPTION
**Pull request by ebenahar**
Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

File: conf/ocsci/ocp-4.4-config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+# Use this config for installation of OCP 4.3 cluster
```
Comment (jilju) [inline conf/ocsci/ocp-4.4-config.yaml]:
```suggestion
# Use this config for installation of OCP 4.4 cluster
```

File: conf/ocsci/ocp-4.4-config.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+RUN:
+  client_version: '4.4.0-0.nightly'
+DEPLOYMENT:
+  installer_version: "4.4.0-0.nightly"
+ENV_DATA:
+  vm_template: 'rhcos-4.3.0-x86_64-vmware'
```
Comment (petr-balogh) [inline conf/ocsci/ocp-4.4-config.yaml]: We should also use the 4.4 rhcos image.  We can take one from https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.4.0-0.nightly-2020-02-13-103342/

File: conf/ocsci/ocp-4.4-config.yaml (hunk 1 above)
Comment (ebenahar) [reply conf/ocsci/ocp-4.4-config.yaml]: done

File: conf/ocsci/ocp-4.4-config.yaml (hunk 2 above)
Comment (ebenahar) [reply conf/ocsci/ocp-4.4-config.yaml]: changed to 'rhcos-4.4.0-x86_64-vmware'

**Comment by ebenahar** [comment]
verified here https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/4719/console

**Pull request merged**